core/src/Pos/Core/Common/AddrStakeDistribution.hs

```diff
@@ -35,9 +35,9 @@ data AddrStakeDistribution
     -- has a value, portion of this value is stake). The constructor
     -- is unsafe because there are some predicates which must hold:
     --
-    -- • the sum of portions must be @maxBound@ (basically 1);
-    -- • all portions must be positive;
-    -- • there must be at least 2 items, because if there is only one item,
+    --   the sum of portions must be @maxBound@ (basically 1);
+    --   all portions must be positive;
+    --   there must be at least 2 items, because if there is only one item,
```

Comment:
is your editor automatically deleting all non-ASCII characters?

Reply:
Looks like my emacs session was holding on to some non-IOHK version of `stylish-haskell`. Nothing re-starting emacs can't fix!